### ISSUE TITLE
Sourcemap not working with awesome-typescript-loader

### ISSUE THREAD

**Issue by thekip** [huston007/ng-annotate-loader#24]
```js
loaders: [
      {
        test: /\.[tj]s$/,
        loaders: ['ng-annotate', 'awesome-typescript-loader'],
        exclude: /node_modules/,
      },
```
i'm using ` devtool: 'cheap-module-source-map',` option. 

And see in console: 

> Failed to parse SourceMap: ...

awesome-typescript-loader is version 1.1.1, with typescript 1.8

**Comment by huston007** [comment]
@thekip does this work without 'ng-annotate' loader?

**Comment by thekip** [comment]
Surely

**Comment by thekip** [comment]
Actually it's not so easy to reproduce, i've created a small application and bug is not appear. 

Also i turned `cheap-module-source-map` to `inline-cheap-module-source-map` and looks like now it works. 

I will research more.

**Comment by thekip** [comment]
Looks like now it works  with `cheap-module-source-map` as well. I don't know what was the problem. 

But now has another problem, file listed twice in source maps, and source map weight is twice bigger. 

Symptoms looks same as in this issue: https://github.com/huston007/ng-annotate-loader/issues/23 :

> "webpack:///./index.js",
"webpack:///./index.js?2645",
"webpack:///../~/angular/index.js",
"webpack:///../~/angular/index.js?08f2",
"webpack:///../~/angular/angular.js",
"webpack:///../~/angular/angular.js?58d0",

But in my case SourceMaps still work.

**Comment by LucasThielmann** [comment]
Can confirm that this commit fixed the issue for me. What's the status on a npm release of 0.1.2?

**Comment by huston007** [comment]
Glad to hear! Let's publish it in a next few hours then.

**Comment by huston007** [comment]
@LucasThielmann Published as 0.2.0
